apps/angular/src/app/anime-dashboard/anime-dashboard.component.html

```diff
@@ -0,0 +1,44 @@
+<table mat-table [dataSource]="(allAnime$ | async) ?? []">
+	<!-- Image -->
+	<ng-container matColumnDef="imageSourceURL">
+		<th mat-header-cell *matHeaderCellDef>Image</th>
+		<td mat-cell *matCellDef="let element">
```

Comment:
Since we're working with anime we can give a more meaningful names to cell definitions
```suggestion
		<td mat-cell *matCellDef="let anime">
```